Scanning images: 100%|████████████████████| 1458/1458 [00:00<00:00, 5285.28it/s]
Scanning labels PCB-upright-21/train/labels.cache3 (1206 found, 0 missing, 252 empty, 0 duplicate, for 1458 images): 1458it [00:00, 9800.58it/s]
Scanning images: 100%|██████████████████████| 177/177 [00:00<00:00, 4651.64it/s]
Scanning labels PCB-upright-21/valid/labels.cache3 (177 found, 0 missing, 0 empty, 0 duplicate, for 177 images): 177it [00:00, 8235.99it/s]
Images sizes do not match. This will causes images to be display incorrectly in the UI.
Image sizes 640 train, 640 test
Using 4 dataloader workers
Logging results to runs/train/yolor_p644
Starting training for 1500 epochs...
     Epoch   gpu_mem       box       obj       cls     total   targets  img_size
  0%|                                                    | 0/73 [00:00<?, ?it/s]
Traceback (most recent call last):
  File "train.py", line 637, in <module>
    train(hyp, opt, device, tb_writer, wandb)
  File "train.py", line 346, in train
    pred = model(imgs)  # forward
  File "/scratch/anaconda3/envs/wvgPCB/lib/python3.7/site-packages/torch/nn/modules/module.py", line 727, in _call_impl
    result = self.forward(*input, **kwargs)
  File "/scratch/WVG/yolor/models/models.py", line 543, in forward
    return self.forward_once(x)
  File "/scratch/WVG/yolor/models/models.py", line 604, in forward_once
    x = module(x)
  File "/scratch/anaconda3/envs/wvgPCB/lib/python3.7/site-packages/torch/nn/modules/module.py", line 727, in _call_impl
    result = self.forward(*input, **kwargs)
  File "/scratch/anaconda3/envs/wvgPCB/lib/python3.7/site-packages/torch/nn/modules/container.py", line 117, in forward
    input = module(input)
  File "/scratch/anaconda3/envs/wvgPCB/lib/python3.7/site-packages/torch/nn/modules/module.py", line 727, in _call_impl
    result = self.forward(*input, **kwargs)
  File "/scratch/anaconda3/envs/wvgPCB/lib/python3.7/site-packages/torch/nn/modules/conv.py", line 423, in forward
    return self._conv_forward(input, self.weight)
  File "/scratch/anaconda3/envs/wvgPCB/lib/python3.7/site-packages/torch/nn/modules/conv.py", line 420, in _conv_forward
    self.padding, self.dilation, self.groups)
RuntimeError: CUDA out of memory. Tried to allocate 30.00 MiB (GPU 0; 7.79 GiB total capacity; 6.50 GiB already allocated; 37.94 MiB free; 6.51 GiB reserved in total by PyTorch)